### DISCUSSION TITLE
functional-programing/intro

### DISCUSSION THREAD

**Discussion by giscus** [sunface/rust-by-practice#503] [Book Comments]
# functional-programing/intro

Learning Rust By Practice, narrowing the gap between beginner and skilled-dev with challenging examples, exercises and projects.

https://practice-zh.course.rs/functional-programing/intro.html

**Comment by Mactep1** [comment]
催更！

**Comment by CoderJackZhu** [comment]
支持

**Comment by StarryCode-Lang** [comment]
求更！
